build.gradle.kts

```diff
@@ -1,113 +0,0 @@
-import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
Why go from kotlin DSL for gradle to groovy?

Reply:
Just as in the comment below - the probability of frequent changes in build process this month will be quite high, so we are trying to have as simple construct as possible. Don't be put off by the fact that your PR was taken and then - seemingly - put away - actually it was the first stage of waterfall we all going to experience - and what you've did help us to understand in what direction we want to proceed.

Reply:
Ok.  Thank you for the explanation.